packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -346,6 +349,32 @@ class JavaGenerator extends Generator {
         }
     }
 
+    private emitPackageInfo(mod: spec.Assembly) {
+        if (!mod.docs) { return; }
+
+        const packageName = this.getNativeName(mod, undefined);
+        const packageInfoFile = this.toJavaFilePath(mod.name + '.package-info');
+        this.code.openFile(packageInfoFile);
+        this.code.line('/**');
+        if (mod.readme) {
+            for (const line of md2html(mod.readme.markdown).split('\n')) {
+                this.code.line(` * ${line}`);
+            }
+            this.code.line(' *');
+        }
+        if (mod.docs.deprecated) {
+            this.code.line(` * @deprecated ${mod.docs.deprecated}`);
+        } else if (mod.docs.stability) {
+            this.code.line(` * @stability ${mod.docs.stability}`);
+        }
+        this.code.line(' */');
+        if (mod.docs.deprecated) {
+            this.code.line('@Deprecated');
```

Comment:
Can you put the message here?

Reply:
The message is present two lines above in the `@deprecated` Javadoc field. This one is the annotation, which - again - is not parameterized.